cli.js

```diff
@@ -15,7 +15,7 @@
 	}
 
 	require("v8-compile-cache");
-	const ErrorHelpers = require("./errorHelpers");
+	const ErrorHelpers = require("./utils/errorHelpers");
```

Comment:
I don't understand this change. It is wrong, it should be `webpack-cli-utils` right?

Reply:
No, this is outer-scope utils for the core CLI files. Maybe naming convention is bad?